extensions/tokenizers/src/main/java/ai/djl/huggingface/translator/TextEmbeddingTranslator.java

```diff
@@ -193,7 +212,7 @@ private NDArray processEmbedding(NDManager manager, NDList list, Encoding encodi
                             .get(0);
         }
         if (normalize) {
-            embedding = embedding.normalize(2, 0);
+            embedding = embedding.normalize(2, 1);
```

Comment:
This should also be -2, this function should work for both batch and unbatched case.